static/app/locale.tsx

```diff
@@ -41,7 +40,7 @@ export function toggleLocaleDebug() {
 /**
  * Global Jed locale object loaded with translations via setLocale
  */
-let i18n: any = null;
+let i18n: Jed | null = null;
```

Comment:
it seems that even with the types, i18n is still `any` because Jed has no installed types 

![image](https://user-images.githubusercontent.com/29228205/149722911-8de94539-92f9-4ce7-b6d4-83dbf7be91df.png)

Reply:
Oh yes, very good catch!

Reply:
@priscilawebdev I checked and it looks like Jed pkg has no types and the support doesnt seem active anymore...

Reply:
We could always add type definitions for `Jed` to our type library.